yarn.lock

```diff
@@ -14891,7 +14891,7 @@ rxjs@^5.5.2:
   dependencies:
     symbol-observable "1.0.1"
 
-rxjs@^6.4.0, rxjs@~6.5.2:
+rxjs@^6.4.0:
```

Comment:
Is rxjs used somewhere else in the repo?

Reply:
Just discovered this neat command: https://yarnpkg.com/lang/en/docs/cli/why/

eslint and storybook uses it:

<img width="753" alt="Screen Shot 2019-11-19 at 10 18 27 AM" src="https://user-images.githubusercontent.com/5016731/69164679-073d3980-0ab6-11ea-926b-6db62dff6073.png">